change/@azure-msal-node-2020-10-29-13-30-16-msal-node-set-logger.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

For msal-node these still fall under "prerelease" if I'm not mistaken.
```suggestion
  "type": "prerelease",
```

---

Oh, good to know! Thanks!